packages/nextjs/server.js

```diff
@@ -0,0 +1,16 @@
+// TODO add instance / constructor export for clerkApi
+
+let exportLib = {};
+
+if (process.env.NEXT_RUNTIME === 'edge') {
```

Comment:
❓ Is this variable available on Vercel Paas?

Reply:
Yes, it should be set on any PaaS running Next.js:
https://nextjs.org/docs/advanced-features/react-18/switchable-runtime#global-runtime-option

From past research, we believe it's available in both getServerSideProps and API routes, and regardless of whether the runtime is set globally or on a per-API route basis.

Reply:
Thanks for checking this.